site/en/docs/extensions/reference/userscripts/index.md

```diff
@@ -0,0 +1,102 @@
+---
+api: userScripts
+---
+{% Aside 'note' %}
+This API is currently available in Chrome 120 beta.
+{% endAside %}
+## Manifest
+
+To use the `chrome.userScripts` API, add the `"userScripts"` permission to your manifest.json and `"host_permissions"` for sites you want to run scripts on.
+
+```json/9-15
+{
+  "name": "User script test extension",
+  "manifest_version": 3,
+  "minimum_chrome_version": "118",
+  "permissions": [
+    "userScripts"
+  ],
+  "host_permissions": [
+    "*://requested.com/*"
```

Comment:
I see what you were trying to do, but I think it's usually best to avoid domains which someone might later buy?

```suggestion
    "*://example.com/*"
```